mllib/src/test/scala/org/apache/spark/mllib/classification/LogisticRegressionSuite.scala

```diff
@@ -38,7 +38,7 @@ object LogisticRegressionSuite {
     scale: Double,
     nPoints: Int,
     seed: Int): java.util.List[LabeledPoint] = {
-    seqAsJavaList(generateLogisticInput(offset, scale, nPoints, seed))
+    seqAsJavaListConverter(generateLogisticInput(offset, scale, nPoints, seed)).asJava
```

Comment:
Oh! I seem to have overlooked this entirely, that the point of `JavaConverters` is that the "xAsY" functions are still implicit. Well this will clean this up a lot. Thanks for pointing that out before I got too far. It won't be hard to touch this up in the changes so far.